community-call/20230223.md

```diff
@@ -0,0 +1,87 @@
+# SCS Tech Weekly 2023-02-23
+
+## Meta
+
+- Minutes: Jonas
+
+## Team updates
+
+### Team IaaS
+
+Minutes: https://github.com/SovereignCloudStack/minutes/blob/main/iaas/20230222.md
+
+- Refinement and progress discussion of ongoing things
+- Focus on upcoming R4
+- Flavor naming concept conflict (OpenShift putting flavors into annotations)
+- Include nameserver in OSISM
+- Magnum Cluster-API based driver
+
+#### Overflow slot appointment
+
+- Discussed open issues for the upcoming release
+- Will have to make sure that a future tender can work on the openstack image manager
```

Comment:
This is not correct and was likely not told to be understood clearly.

There are two tenders that are already running (VP01 and VP06d) where work regarding the OpenStack Image Manager is suiteable. The info was that we will make sure the work is being done as part of those.

Reply:
Fixed hopefully, please verify!

Reply:
I adjusted it, apparently I was not clear ;)